dev-db/mysqltuner/mysqltuner-1.7.15.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN=MySQLTuner-perl
+
+DESCRIPTION="MySQLTuner is a high-performance MySQL tuning script"
+HOMEPAGE="http://www.mysqltuner.com/"
+SRC_URI="https://github.com/major/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	virtual/mysql
+	dev-lang/perl
+	virtual/perl-Getopt-Long"
+
+DEPEND="${RDEPEND}"
+
+PATCHES=(
+	# Fix password detecting under MySQL 5.7 upstream #442 #427 #441
+	"${FILESDIR}/${P}-password.patch" )
```

Comment:
Please also try getting this patch applied into upstream. It's not a showstopper now, but it's always nicer to drop carrying around patches forever.

Reply:
Upstream aware of that issue ( #442 #427 #441 ) but haven't been fixed yet. I implemented quick fix.